package.json

```diff
@@ -13,7 +13,8 @@
     "install": "node ./preinstall.js && node-gyp rebuild",
     "build": "node-gyp rebuild",
     "start": "node .",
-    "test": "echo \"Error: no test specified\" && exit 0"
+    "test": "NODE_ENV=test mocha --watch tests/unit/**/*.test.js",
+    "test:ci": "NODE_ENV=test mocha tests/unit/**/*.test.js"
```

Comment:
Does this run on CI? Let's see...

I've found just booting on CI can be nontrivial because of the GL requirement.

Reply:
- https://github.com/webmixedreality/exokit/blob/master/appveyor.yml
- https://github.com/webmixedreality/exokit/blob/master/.travis.yml

Reply:
Forgot to add `cross-env`. Pushed a fix, will see how it goes. I don't think the unit tests will invoke GL, else we can stub it out in test mode. We do the same for A-Frame.